src/core/Icon/Icon.md

```diff
@@ -7,8 +8,31 @@ import { Icon } from 'suomifi-ui-components';
   <Icon
     icon="login"
     ariaLabel="Login here"
-    className="my-icon--test"
+    className="my-icon-class"
```

Comment:
Does class name have any effect? Should we rather leave it out or mention it in the docs?

Reply:
I don't think so. Some of the examples seem to have classNames defined "just because". I'll remove it to make the example clearer.